### DISCUSSION TITLE
Review and ATS Score for my resume

### DISCUSSION THREAD

**Discussion by TechSpire34** [recodehive/Stackoverflow-Analysis#517] [Q&A] [answered]
Review and ATS Score for my resume

**Comment by sandipanrakshit34** [comment, marked as answer]
Hello TechSpire,

Thank you for reaching out and sharing your resumes. I appreciate your proactive approach to refining your job applications.

I'll review both documents thoroughly for their respective profiles—Business Analyst and Software Engineering. I'll provide feedback on content, structure, and ATS optimization to ensure your resumes align with industry standards and effectively highlight your skills and achievements.

You can expect my insights shortly. Meanwhile, if there are specific job descriptions you're targeting, please share them with me so I can tailor my feedback accordingly.

Best regards,
Sandipan Rakshit

**Comment by Pranjal-88** [comment]
Hey 
I would like to look up and review your resume !!
